### PR TITLE
Add Homework

### PR DESCRIPTION
**Pull request by VladislavBr**

### CODE REVIEW

**Review by Roman-Derback — DISMISSED** [review]
a lot of unnecessary files
but nice

File: Projects/Team2/Vlad Borisovets/Animals/Animals/Fish.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+		public Fish()
+		{
+
+		}
+		public Fish(int yearOfBirth, string color, string kind= "Mackerel", string species= "Amberjack") :base(yearOfBirth, color)
+		{
+			this.kind = kind;
+			this.species = species;
+		}
+	}
```
Comment (romanukhaav) [inline Projects/Team2/Vlad Borisovets/Animals/Animals/Fish.cs:34]: it seems like your fishes can bark like dogs :)

File: Projects/Team2/Vlad Borisovets/Animals/Animals/Program.cs [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+         відсортуйте інформацію за видом (Species)
+         збережіть колекцію у файл
+         реалізуйте опрацювання виключень (Exceptions)
+         сереалізуйте (Serialize) колекцію в XML файл
+         десереалізуйте колекцію з XML файла
+         напишіть unit тести до реалізованих функцій*/
+		public static List<Animals> animals = new List<Animals>();
+		static void Main(string[] args)
+		{
+			//var animals = new List<Animals>();
```
Comment (romanukhaav) [inline Projects/Team2/Vlad Borisovets/Animals/Animals/Program.cs]: please remove unnecessary comments

File: Projects/Team2/Vlad Borisovets/Animals/Animals/Program.cs [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+		public static void mods()
+		{
+			Console.WriteLine("Select mode:\n0 - Add new animal\n1 - Add new Fish\n2 - Show all animals\n3 - Show all Fish\n4 - Fish older than 3 years and sorted by species" +
+				"\n5 - Write to file 'Animals.txt'\n6 - Serialize and Deserialize list cars to 'Animals.xml'");
+			int mode = Int32.Parse(Console.ReadLine());
+			switch (mode)
+			{
+				case 0:
+					GettersAnimal();
+					mods();
```
Comment (romanukhaav) [inline Projects/Team2/Vlad Borisovets/Animals/Animals/Program.cs:58]: nice decision - it is like some kind of recursion

File: Projects/Team2/Vlad Borisovets/Animals/Animals/Program.cs [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+			Console.WriteLine("Write year of bith your animal: ");
+			int YearOfBirth = Int32.Parse(Console.ReadLine());
+			Console.WriteLine("Write color your animal: ");
+			string Color = Console.ReadLine();
+			animals.Add(new Animals(YearOfBirth, Color));
+		}
+		//Add new Fish
+		public static void GettersFish()
+		{
+			Console.WriteLine("Write year of bith fish: ");
```
Comment (romanukhaav) [inline Projects/Team2/Vlad Borisovets/Animals/Animals/Program.cs]: This method could cause errors. Please rewrite this by using try/catch or TryParse()

**Review by romanukhaav — APPROVED** [review]
Great!

**Pull request merged**